.github/workflows/demo-preview-deploy.yml

```diff
@@ -0,0 +1,101 @@
+
+
+name: Demo Preview
+
+on:
+  pull_request:
```

Comment:
Shouldn't this be ?
```suggestion
  pull_request_target:
```

Reply:
I think we want to restrict it running on forks, so keeping it as pull_request only is what I'd prefer for now, might change this in the future though